src/app/containers/Timestamp/index.jsx

```diff
@@ -5,16 +5,23 @@ import relativeTime from './relativeTimestamp';
 import {
   longNumeric,
   shortAlphaNumeric,
+  alphaNumericDatetime,
   isValidDateTime,
   formatUnixTimestamp,
   isTenHoursAgoOrLess,
+  isTwentyFourHoursAgoOrLess,
 } from './timestampUtilities';
 import { GridItemConstrainedMedium } from '../../lib/styledGrid';
 
+const formatType = timestamp =>
+  isTwentyFourHoursAgoOrLess(timestamp)
```

Comment:
I tried to replace this with moment logic but it seems as if the 24 hour check is more accurate, both [today.diff(timestamp) <= 0](https://stackoverflow.com/questions/21284312/moment-js-check-if-a-date-is-today-or-in-the-future) and [today.isSame(timestamp, 'd')](https://stackoverflow.com/questions/21284312/moment-js-check-if-a-date-is-today-or-in-the-future) cause the related tests to fail.